tools/lkl/lib/posix-host.c

```diff
@@ -311,6 +312,41 @@ static long _gettid(void)
 #endif
 }
 
+static unsigned int cpuinfo_get(char *buffer, unsigned int buffer_len)
+{
+	int len;
+
+	len = snprintf(buffer, buffer_len,
```

Comment:
Perhaps leave a comment here that this is hard coded for now? Eventually this should come from the host with values overridden by the enclave_config.

Reply:
Done.